src/liquid/components/ld-tabs/test/ld-tabs.e2e.ts

```diff
@@ -139,176 +183,148 @@ describe('ld-tabs', () => {
   })
 
   describe('mode', () => {
-    it(`ghost`, async () => {
-      const page = await getPageWithContent(getTabsHTML('mode="ghost"'))
-      const results = await page.compareScreenshot()
-      expect(results).toMatchScreenshot()
-    })
-
-    it(`ghost hover`, async () => {
-      const page = await getPageWithContent(getTabsHTML('mode="ghost"'))
-      await page.hover('ld-tab:nth-of-type(2)')
-      const results = await page.compareScreenshot()
-      expect(results).toMatchScreenshot()
-    })
-
-    it(`ghost hover selected`, async () => {
-      const page = await getPageWithContent(getTabsHTML('mode="ghost"'))
-      await page.hover('ld-tab:nth-of-type(1)')
-      const results = await page.compareScreenshot()
-      expect(results).toMatchScreenshot()
-    })
-
-    it(`ghost hover disabled`, async () => {
-      const page = await getPageWithContent(getTabsHTML('mode="ghost"'))
-      await page.hover('ld-tab:nth-of-type(3)')
-      const results = await page.compareScreenshot()
-      expect(results).toMatchScreenshot()
-    })
-
-    it(`ghost focus`, async () => {
-      const page = await getPageWithContent(getTabsHTML('mode="ghost"'))
-      await page.keyboard.press('Tab')
-      await page.keyboard.press('ArrowRight')
-      const results = await page.compareScreenshot()
-      expect(results).toMatchScreenshot()
-    })
-
-    it(`ghost focus selected`, async () => {
-      const page = await getPageWithContent(getTabsHTML('mode="ghost"'))
-      await page.keyboard.press('Tab')
-      const results = await page.compareScreenshot()
-      expect(results).toMatchScreenshot()
-    })
-
-    it(`ghost focus disabled`, async () => {
-      const page = await getPageWithContent(getTabsHTML('mode="ghost"'))
-      await page.keyboard.press('Tab')
-      await page.keyboard.press('ArrowRight')
-      await page.keyboard.press('ArrowRight')
-      await page.keyboard.press('ArrowRight')
-      const results = await page.compareScreenshot()
-      expect(results).toMatchScreenshot()
-    })
-
-    it(`ghost active`, async () => {
-      const page = await getPageWithContent(getTabsHTML('mode="ghost"'))
-      await page.keyboard.press('Tab')
-      await page.keyboard.press('ArrowRight')
-      await page.keyboard.down('Space')
-      const results = await page.compareScreenshot()
-      expect(results).toMatchScreenshot()
-    })
-
-    it(`ghost active selected`, async () => {
-      const page = await getPageWithContent(getTabsHTML('mode="ghost"'))
-      await page.keyboard.press('Tab')
-      await page.keyboard.down('Space')
-      const results = await page.compareScreenshot()
-      expect(results).toMatchScreenshot()
-    })
+    for (const mode of ['ghost', 'brand-color', 'floating']) {
+      it(`${mode}`, async () => {
```

Comment:
I'd suggest s/th like this:

```suggestion
    for (const mode of ['ghost', 'brand-color', 'floating']) {
      describe(mode, () => {
        it('default', async () => {
```

Reply:
Maybe next time.